broker/setup_broker/setup_gcp.py

```diff
@@ -253,48 +256,49 @@ def setup_bigquery(survey='ztf', testid='test', teardown=False, versiontag="v3_3
 
-    (datasets, table_data) = _resources('BQ', survey=survey, testid=testid, versiontag=versiontag)
+    (datasets, table_data) = _resources("BQ", survey=survey, testid=testid, versiontag=versiontag)
     bigquery_client = bigquery.Client(location=region)
 
     for dataset, tables in datasets.items():
         if teardown:
             # Delete dataset
-            kwargs = {'delete_contents': True, 'not_found_ok': True}
+            kwargs = {"delete_contents": True, "not_found_ok": True}
             bigquery_client.delete_dataset(dataset, **kwargs)
-            print(f'Deleted dataset {dataset}')
+            print(f"Deleted dataset {dataset}")
         else:
             # Create dataset
             try:
                 bigquery_client.get_dataset(dataset)
                 print(f"Skipping existing dataset: {dataset}")
             except NotFound:
                 bigquery_client.create_dataset(dataset)
-                print(f'Created dataset: {dataset}')
+                print(f"Created dataset: {dataset}")
 
             # create the tables
             # use the bq CLI so we can create the table from a schema file
             for table in tables:
                 try:
-                    table_id = f'{PROJECT_ID}.{dataset}.{table}'
+                    table_id = f"{PROJECT_ID}.{dataset}.{table}"
                     bigquery_client.get_table(table_id)
-                    print(f'Skipping existing table: {table_id}.')
+                    print(f"Skipping existing table: {table_id}.")
                 except NotFound:
                     table_id = f"{PROJECT_ID}:{dataset}.{table}"
                     schema = f"templates/bq_{survey}_{table}_schema"
                     if table_data.get(table):
                         # make the table and load the data
                         data = f"templates/bq_{survey}_{table_data[table]}.csv"
                         flags = f"--schema={schema}.json --source_format=CSV"
-                        bqload = f'bq load {flags} {table_id} {data}'
+                        bqload = f"bq load {flags} {table_id} {data}"
                         out = subprocess.check_output(shlex.split(bqload))
-                        print(f'{out}')  # should be a success message
+                        print(f"{out}")  # should be a success message
                     else:
                         # no data to load, just create the table
-                        bqmk = f'bq mk --table {table_id} {schema}.json'
+                        bqmk = f"bq mk --table {table_id} {schema}.json"
                         out = subprocess.check_output(shlex.split(bqmk))
-                        print(f'{out}')  # should be a success message
+                        print(f"{out}")  # should be a success message
+
 
-def setup_dashboard(survey='ztf', testid='test', teardown=False) -> None:
+def setup_dashboard(survey="ztf", testid="test", teardown=False) -> None:
```

Comment:
Go ahead and delete this whole function plus the related private functions and any other variables that would then be obsolete. I'm pretty sure it doesn't even work anymore but just throws an error. It would be great to have a dashboard, but this particular implementation was too fiddly to be sustainable. That should also take care of all the Codacy errors as well.